server-spi-private/src/main/java/org/keycloak/models/utils/RepresentationToModel.java

```diff
@@ -1606,8 +1605,7 @@ public static ClientScopeModel createClientScope(KeycloakSession session, RealmM
         if (resourceRep.getProtocol() != null) clientScope.setProtocol(resourceRep.getProtocol());
         if (resourceRep.getProtocolMappers() != null) {
             // first, remove all default/built in mappers
-            Set<ProtocolMapperModel> mappers = clientScope.getProtocolMappers();
-            for (ProtocolMapperModel mapper : mappers) clientScope.removeProtocolMapper(mapper);
+            clientScope.getProtocolMappersStream().forEach(clientScope::removeProtocolMapper);
```

Comment:
Same: Is there a chance that a concurrent modification exception is thrown here?

Reply:
Same as above